internal/engine/sqlite/parser/SQLiteParser.g4

```diff
@@ -76,6 +76,7 @@ alter_table_stmt:
             | COLUMN_? old_column_name = column_name TO_ new_column_name = column_name
         )
         | ADD_ COLUMN_? column_def
+        | DROP_ COLUMN_? column_name
```

Comment:
Added this support to the parser required a region, which explains a lot of the size of this PR.